File: Programme/WMS_Java/WarehouseManagementSystem/src/data/Lager.java
package data;

public class Lager {

    private String lId;
    private String name;
    private String adresse;

    public Lager(String id, String name, String adresse) {
        setlId(id);
        setName(name);
        setAdresse(adresse);
    }

    public String getlId() {
        return lId;
    }
    private void setlId(String lId) {
        this.lId = lId;
    }

    public String getName() {
        return name;
    }
    private void setName(String name) {
        this.name = name;
    }

    public String getAdresse() {
        return adresse;
    }
    private void setAdresse(String adresse) {
        this.adresse = adresse;
    }
}

File: Programme/WMS/WMS_CL/Lager/Lager.cs
public class Lager
{
    private string _myID = "L";

    public Lager(string LID, string Name, string Adresse)
    {
        _myID = LID;
        this.Name = Name;
        this.Adresse = Adresse;
    }

    public Lager(int LID, string Name, string Adresse)
    {
        _myID += LID;
        this.Name = Name;
        this.Adresse = Adresse;
    }

    public string LID
	{
		get
        {
            return _myID;
        }
	}

	public string Name
	{
		get;
		set;
	}

	public string Adresse
	{
		get;
		set;
	}

    public override string ToString()
    {
        return "Lager " + Name + " in " + Adresse;
    }
}

